.env.testnet.staging

```
@@ -0,0 +1,25 @@
+# This file contains the environment variables for the CR staging testnet environment
+NEXT_PUBLIC_ENV=testnet
+NEXT_PUBLIC_RIF_WALLET_SERVICES=https://frontend.testnet.bim.dao.rif.technology/cors_bypass
+NEXT_PUBLIC_PROXY_DESTINATION=https://rws.app.rootstockcollective.xyz
+NEXT_PUBLIC_EXPLORER=https://explorer.testnet.rootstock.io
+NEXT_PUBLIC_BUILD_ID=
+
+NEXT_PUBLIC_RIF_ADDRESS=0x19f64674d8a5b4e652319f5e239efd3bc969a1fe
+NEXT_PUBLIC_STRIF_ADDRESS=0x4861198e9A6814EBfb152552D1b1a37426C54D23
+NEXT_PUBLIC_GOVERNOR_ADDRESS=0x2109FF4a9D5548a21F877cA937Ac5847Fde49694
+NEXT_PUBLIC_EA_NFT_ADDRESS=0x979deF73ec80B8AE24Ae46765b81D9aF7b1C9327
+NEXT_PUBLIC_MULTICALL_ADDRESS=0xcA11bde05977b3631167028862bE2a173976CA11
+NEXT_PUBLIC_GRANTS_BUCKET_ADDRESS=0xfaca664c661af7e0e630c8f92b401012cd2a30ef
+NEXT_PUBLIC_GRANTS_ACTIVE_BUCKET_ADDRESS=0x2217E4d3Ae0A6E30075D1B5a7b8C1520E8009f49
+NEXT_PUBLIC_GROWTH_BUCKET_ADDRESS=0x420f39968009a0e8693a0e1bba633cd99eaffb9b
+NEXT_PUBLIC_GENERAL_BUCKET_ADDRESS=0x72Ed7d7b7835Ad62B1f9b6280bAd62618aA71461
+NEXT_PUBLIC_CHAIN_ID=31
+
+# CR-related env variables
+NEXT_PUBLIC_SIMPLIFIED_REWARD_DISTRIBUTOR_ADDRESS=0x4e84FCc953dE129C6C47c5B0AD7E57B226093Ae1
```

```suggestion
NEXT_PUBLIC_SIMPLIFIED_REWARD_DISTRIBUTOR_ADDRESS=0x4447EC87854651ECdce3E91e581c34641C91eC46
```

See https://github.com/rsksmart/bim-dao-frontend/pull/71/files